resources/service-catalog-addons/charts/service-catalog-ui/templates/brokers-microfrontend.yaml

```
@@ -10,6 +10,7 @@ spec:
   category: Configuration
   placement: namespace
   viewBaseUrl: 'https://brokers.{{ js .Values.global.ingress.domainName }}'
+  preloadUrl: 'https://brokers.{{ js .Values.global.ingress.domainName }}/#preload'
```

brokers, catalog and instances views use 'path routing' strategy to handle page navigation. you need to get rid of `#` to make it work :)
```suggestion
  preloadUrl: 'https://brokers.{{ js .Values.global.ingress.domainName }}/preload'
```